examples/grafana-integration/docker-compose.yaml

```diff
@@ -0,0 +1,51 @@
+version: '2'
+services:
+  grafana:
+    image: grafana/grafana:7.1.4
+    ports:
+      - '3000:3000'
+    volumes:
+      - ./datasources.yaml:/etc/grafana/provisioning/datasources/datasources.yaml
+    logging:
+      driver: loki
+      options:
+        loki-url: 'http://localhost:3100/api/prom/push'
```

Comment:
Shouldn't this be `loki:3100`?

Reply:
The docker driver runs in the host network namespace.  So localhost is correct.  I just learned this recently for a little demo I put together.

Reply:
I am not sure this is universal behavior. I'd say it's clearer to refer to the host name of the service than assume that localhost is correctly connected.

Reply:
Right, but "loki" won't resolve in the host network namespace and the driver won't be able to send logs.  The port mapping on the loki service allows the docker logging driver to refer to localhost:3100 and successfully send logs to loki.

This does work consistently in Linux.  Unsure about OSX.

Reply:
why wouldn't `loki` resolve, but `jaeger` in L20 does?

Reply:
because the logging driver is running on the host, but the containers are running on the docker bridge.

Reply:
That is correct. 
I was experimenting with different configs, and this came out as the most portable and consistent solution. An other one could be using `http://host.docker.internal:3100`, which can work on OSX and Windows, but only recently (?) on Linux, so I did go with the former one.
I can also confirm that it's working fine in both Linux and in OSX, I believe it should work on Windows as well, but currently I don't have access to any.